### ISSUE TITLE
Northern Rebirth issues

### ISSUE THREAD

**Issue by gfgtdf** [wesnoth/wesnoth#2569]
Copied from the forum https://forums.wesnoth.org/viewtopic.php?f=5&p=623812#p623812

> I'm playing the campaign NORTHERN REBIRTH with BfW 1.13.11 for Windows, difficult level, and I have met some little drawbacks;
I,m not capable to use the appropriate github page so I report them here, I'm sorry  :augh: 
>
>Scenario INFESTED CAVES
> 
>After Malifor is defeated appear this message:
> 
>[code]<Lua> repeated [endlevel] execution ignoring[/code]
> 
>Scenario OLD FRIEND
> 
>In the dialogues Rakshas has not his profile image but the orcish sovereign portrait
> 
>Scenario GET THE GOLD
> 
>When the orc leader is defeated appear this message:
> 
>
```
<Lua error> game_error:invalid side (9) found in unit definition
stack traceback:
[C]: in field 'create unit'
lua/wml-tags.lua:839: in local 'cmd'
lua/wml-utils.lua:145: in field 'handle_event_commands'
lua/wml-flow.lua:200: in local 'cmd'
lua/wml-utils.lua:145: in field 'handle event_commands'
lua/wml-flow.lua:6: in function <lua/wml-flow.lua:5>
```
> 
> Scenario SHOWDOWN
> 
> Ro'Sothian is the leader of the south faction even if both Elenia and Eryssa are still alive (I'm not sure this is wrong  :? )
>
>My apologies for my english and for my ineptitude to use github

**Comment by sigurdfdragon** [comment]
@Vultraz 
I've been doing a playthrough of NR & preparing some bugfixes, I can add the first two to my forthcoming pr if you'd like. (somehow I missed those)

I'm about to start Get the Gold, so I haven't had a chance to run into the last two yet.

**Comment by Vultraz** [comment]
@sigurdfdragon if you have fixes, by all means push them. I don't intend to work in this for at least another week or so.

**Comment by sigurdfdragon** [comment]
Ok, I'll likely get to all four of the above-mentioned items then.

**Comment by sigurdfdragon** [comment]
closed by #2600
